internal/v1/api.yaml

```diff
@@ -581,6 +612,44 @@ components:
             Array of exactly one image request. Having more image requests in one compose is currently not supported.
         customizations:
             $ref: '#/components/schemas/Customizations'
+    CreateBlueprintRequest:
+      type: object
+      additionalProperties: false
+      required:
+        - name
+        - description
+        - distribution
+        - image_requests
+        - customizations
+      properties:
+        name:
+          type: string
+          example: "My Blueprint"
+          maxLength: 100
+        description:
+          type: string
+          example: "My blueprint description"
+          maxLength: 250
+        distribution:
+          $ref: '#/components/schemas/Distributions'
+        image_requests:
+          type: array
+          minItems: 1
+          maxItems: 1
```

Comment:
We need to support all target types, and that number will probably increase... maybe no max needed?

Reply:
Ditto.

Reply:
I suggest to do this separately after this is merged in both components.

Reply:
This should be different in Blueprint and Compose, here it should be 1:n, but compose stay at 1:1, we'd just make N composes when making composes out of blueprint (rebuilding blueprint images) or do I misunderstand something?

I'm not opposed to do it in follow up, I just don't get why 🤔 but sure, I can propose it myself 👍 I can have contribution!! 😛

Reply:
I don’t understand, this is a datatype from "create compose" API request without any change. I believe that blueprint is just a "template" of what we have today. Therefore, I designed this endpoint in a way that blueprint can be resubmitted "as is" without any changes.

Maybe let’s have a chat on Slack to speed things up, I might be missing something.

Reply:
Oh now I understand, multiple image requests can be in a single blueprint. This is different from what is now. Fixed.